RacingCar/InputChecker.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  InputChecker.swift
+//  RacingCar
+//
+//  Created by nylah.j on 2022/04/30.
+//
+
+import Foundation
+
+struct InputChecker {
+    static func isValid(carCount: Int?) -> Bool {
+        return carCount != nil && carCount! > 0
+    }
+    
+    static func isValid(roundCount: Int?) -> Bool {
```

Comment:
```suggestion
    static func isValidRoundCount(_ count: Int?) -> Bool {
```
위 코멘트처럼 의견드려봅니다 🙇🏻‍♂️